Gemfile.jruby-2.5.lock.release

```diff
@@ -304,7 +304,7 @@ GEM
       logstash-core-plugin-api (>= 1.60, <= 2.99)
       rufus-scheduler
       stud (~> 0.0.22)
-    logstash-input-file (4.1.16)
+    logstash-input-file (4.1.18)
```

Comment:
Version 4.1.18 introduced [a bug](https://github.com/logstash-plugins/logstash-input-file/issues/272) that was only solved in 4.2.0, but 4.1.17 seems harmless and improves documentation:

```suggestion
    logstash-input-file (4.1.17)
```